jwst/associations/lib/rules_level2_base.py

```diff
@@ -894,6 +895,21 @@ def __init__(self):
             sources=['targetid'],
         )
 
+class Constraint_ExtCal(Constraint):
+    """Remove any nis_extcals from the associations, they
+       are NOT to be processed in the pipline!"""
```

Comment:
Technically they *are* processed in the cal pipeline, but only through the first stage (level-2a) of calibration. It's only the later stages (2b and 3) that use ASN's as drivers, hence we don't want them showing up in any ASN's. So to be completely correct, you could change this comment to "... they are NOT to receive level-2b or level-3 processing".

Reply:
The comments are updated.